package.json

```diff
@@ -16,24 +18,41 @@
     "fetch",
     "axios"
   ],
+  "ava": {
+    "extensions": [
+      "ts"
+    ],
+    "require": [
+      "ts-node/register"
+    ]
+  },
   "author": "LifeOmic <development@lifeomic.com>",
   "license": "MIT",
   "devDependencies": {
     "@lifeomic/eslint-plugin-node": "^2.0.1",
+    "@types/lodash.mapvalues": "^4.6.6",
+    "@types/sinon": "^10.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.22.0",
+    "@typescript-eslint/parser": "^4.22.0",
     "ava": "^3.15.0",
     "axios": "0.21.1",
     "coveralls": "^3.1.0",
     "eslint": "^6.0.0",
     "lodash.mapvalues": "^4.6.0",
     "nock": "^11.3.3",
     "nyc": "^15.0.0",
-    "sinon": "^7.2.4"
+    "sinon": "^10.0.1",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.2.4"
   },
   "dependencies": {
-    "form-data": "^2.3.3",
+    "@types/lodash.identity": "^3.0.6",
```

Comment:
Can we move these into the `devDependencies`? I have seen @types in the dependency list when the types are re-exported, but it doesn't seem like all of these types will be exported

Reply:
I've had issues where the type was used internally and not exported, so no I export any types used in lib code.